[FARM-504] Currency formatting update

alexanishchenko88 opened the pull request:
#### 📲 What

Currency formatting has been changed to currency instead of compactCurrency to avoid rounding issues.

#### 🤔 Why
		
We should update this formatting to avoid wrong currency display.
		
#### 🛠 How
		
Just update from from one formatting to another one

#### 👀 See

<img width="50%" alt="Premium User" src="https://user-images.githubusercontent.com/40355600/65416192-9f40de80-ddf7-11e9-9400-f500838c9402.png">
	 
#### ✅ Acceptance criteria

- [ ] Design Review for UI with BA completed. 
- [ ] Manually tested and verified on Android device/emulator (min. Lollipop 5.1)
- [ ] Showcase video / automation test
- [ ] Passing all tests
- [ ] Rebased/merged with latest changes from development and re-tested
- [ ] Removed unsed comments. TODOs must have JIRA for future work.

#### Coding Standards Checklist
- [ ] unit tests 80% coverage on testable code (functions, methods, class)
- [ ] (Architecture) redux state, reducers, actions, middleware defined under lib/redux
- [ ] ui elements defined under lib/ui. 
- [ ] Strings ready for localisation (e.g. defined in lib/utils/strings.dart)
- [ ] Assets (images/icons path) defined in lib/utils/assets.dart
- [ ] Colors defined in lib/utils/colors.dart
- [ ] Dimensions (ie margins padding) defined in lib/utils/dimens.dart
- [ ] TextStyles defined in lib/utils/styles.dart

Recommended Style Guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

#### 🕵️‍♂️ How to test

Notes for QA

#### Reviewers

@farmsmart/amido @farmsmart/wamf

alexanishchenko88 committed:
[FARM-504] Currency formatting update

What:
Currency formatting has been changed to curreny instead of compactCurrency to avoid rounding issues.

Added:
-

Updated:
curreny instead of compactCurrency in TransactionAmount.dart

Deleted:
-

davidfranquet reviewed (changes requested):
Decimals looks fine but the format should be:
More than 999 => X.X**k** or X.X**m** if it's a million.
For example: 1200 => 1.2k
k & m are for EN locale. If you switch to Swahili the 'k' will be 'efta'

Maybe you can solve this applying the Compact currency only for numbers greater than 999

alexanishchenko88 committed:
[FARM-504] Updated to new way of getting format. If greater or equals to 1000 will give compact currency, otherwise just currency

What:
Formatting updated

Added:
-

Updated:
If greater or equals to 1000 will give compact currency, otherwise just currency

Deleted:
-
